### PR TITLE
Add HTTPS setup instructions for IIS Dev scenario

### LINKED ISSUE #5899: Add HTTPS setup instructions for IIS Dev scenario
https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support

https://www.sslshopper.com/article-how-to-create-a-self-signed-certificate-in-iis-7.html

Testing HTTPS apps at dev time is becoming more common, and is now required for 2.1. The templates and tooling do everything for you for IIS Express. However tooling doesn't do anything for you when developing using IIS so 2.1 apps fail.

Following the docs, you set up IIS for http://localhost/WebApplication1, but when the browser launches the default app redirects you to https. If https isn't enabled in IIS you get a generic browser error about failing to connect and it's not clear that your site is even running.

@shirhatti @BillHiebert

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #5899 

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-1.0&branch=pr-en-us-6415)

### CODE REVIEW

**Comment by shirhatti** [comment]
@javiercn In case he has some comments here.

**Comment by javiercn** [comment]
@shirhatti You don't have to worry about HSTS in localhost, we have a list of excluded hosts and there's a setting to add your own exclusions if you are using a custom domain (and modifying your hosts file).

**Comment by guardrex** [comment]
@shirhatti You good here? ... I'll call for the UE review and get this puppy on its way. 🐶 *Ruf Ruf*

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 1]
```diff
@@ -5,71 +5,136 @@ description: Discover support for debugging ASP.NET Core apps when running behin
...
 ## Enable IIS
 
-Enable IIS. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen). Select the **Internet Information Services** checkbox.
+1. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
+1. Select the **Internet Information Services** check box.
 
-![Windows Features showing Internet Information Services checkbox checked as a black square (not a checkmark) indicating that some of the IIS features are enabled](development-time-iis-support/_static/enable_iis.png)
+![Windows Features showing Internet Information Services check box checked as a black square (not a checkmark) indicating that some of the IIS features are enabled](development-time-iis-support/_static/enable_iis.png)
 
 If the IIS installation requires a restart, restart the system.
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]: ~~If the IIS installation requires a restart, restart the system.~~
The IIS installation may require a system restart.

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 2]
```diff
@@ -5,71 +5,136 @@ description: Discover support for debugging ASP.NET Core apps when running behin
...
 ## Configure the project
 
-Create a new launch profile to add development-time IIS support. In Visual Studio's **Solution Explorer**, right-click the project and select **Properties**. Select the **Debug** tab. Select **IIS** from the **Launch** dropdown. Confirm that the **Launch browser** feature is enabled with the correct URL.
+### HTTPS redirection
+
+For a new project, select the check box to **Configure for HTTPS** in the **New ASP.NET Core Web Application** window:
+
+![New ASP.NET Core Web Application window with the Configure for HTTPS check box selected.](development-time-iis-support/_static/new-app.png)
+
+In an existing project, use Https Redirection Middleware in `Startup.Configure` by calling the [UseHttpsRedirection](/dotnet/api/microsoft.aspnetcore.builder.httpspolicybuilderextensions.usehttpsredirection) extension method:
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]: Https --> HTTPS

**Comment by guardrex** [comment]
ty @scottaddie Updates made. :+1:

**Pull request merged**